### DISCUSSION TITLE
Issues using tkvideoPlayer

### DISCUSSION THREAD

**Discussion by mariopinto18** [PaulleDemon/tkVideoPlayer#62] [General]
Sorry, but when i use tkvideoplayer, it occurs an error:
Example:

import tkinter as tk
from tkVideoPlayer import TkinterVideo

root = tk.Tk()
videoplayer = TkinterVideo(master=root, scaled=True)
videoplayer.pack(expand=True, fill="both")
videoplayer.load(r"AnaMoreira.mp4")
videoplayer.play() # play the video
root.mainloop()

![image](https://github.com/user-attachments/assets/4906e872-86e4-4167-b297-636e1367c92e)

Can you help me @PaulleDemon ?

**Comment by PaulleDemon** [comment]
Can you let me know which version of python you are using and did you try with other videos?

**Reply by mariopinto18** [reply]
Hi!
Python version 3.12.8.
Due to the error i updated to 3.13.1, but the error continues. I tried with a lot of different videos...

About a year ago I had done some projects with tkvideoplayer and I never had this problem.

Thanks for your attention! Greatings

**Reply by PaulleDemon** [reply]
@mariopinto18 Hey sorry, I doesn't support 3.11 and above for now. Will update it in Feb or something, because I am a bit busy now. You can use python 3.10

Also, wish you happy new year :)

**Comment by mariopinto18** [comment]
@***@***.***>Thank you for your feedback!

Best regards



[1458206780344_assinatura]
POLITÉCNICO DO PORTO. ESCOLA SUPERIOR DE MEDIA ARTES E DESIGN
——
Mário Pinto
PhD informatics / computer science
uniMAD | ID+ Research Institute for Design Media and Culture
ORCID: 0000-0002-6734-5797

https://about.me/mario.pinto

M  RUA D. SANCHO I, 981. 4480-876 VILA DO CONDE. PORTUGAL
T +351 252 291 700   F +351 252 291 714

________________________________
De: PaulleDemon ***@***.***>
Enviado: 31 de dezembro de 2024 10:19
Para: PaulleDemon/tkVideoPlayer ***@***.***>
Cc: Mário Pinto ***@***.***>; Mention ***@***.***>
Assunto: Re: [PaulleDemon/tkVideoPlayer] Issues using tkvideoPlayer (Discussion #62)

Não costuma receber e-mails de ***@***.*** Saiba por que motivo isto é importante<https://aka.ms/LearnAboutSenderIdentification>

@mariopinto18<https://github.com/mariopinto18> Hey sorry, I doesn't support 3.11 and above for now. Will update it in Feb or something, because I am a bit busy now. You can use python 3.10

Also, wish you happy new year :)
